src/main/java/io/stargate/sgv2/jsonapi/api/configuration/ObjectMapperConfiguration.java

```diff
@@ -33,8 +34,10 @@ private ObjectMapper createMapper() {
         .enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS)
         // case insensitive enums, so "before" will match to "BEFORE" in an enum
         .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
+        // Verify uniqueness of JSON Object properties
+        .enable(StreamReadFeature.STRICT_DUPLICATE_DETECTION)
         // Prevent use of Engineering Notation with trailing zeroes:
-        .enable(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN)
+        .enable(StreamWriteFeature.WRITE_BIGDECIMAL_AS_PLAIN)
```

Comment:
StreamWriteFeature/JsonWriteFeature will replace JsonGenerator.Feature; same behavior in Jackson 2.x but latter removed from 3.0